pkg/cli/app.go

```diff
@@ -0,0 +1,89 @@
+package cli
+
+import (
+	"os"
+
+	"github.com/urfave/cli/v2"
+)
+
+// App is the main structure of app cli application. It should be created with the cli.NewApp() function.
+type App struct {
```

Comment:
We should explain that it's a wrapper for `urfave`'s `cli.App` struct, and explain what we're adding here that's different.

Reply:
Fair point, explained.